tests/ci/pr_info.py

```diff
@@ -171,7 +169,11 @@ def __init__(
                     response_json = user_orgs_response.json()
                     self.user_orgs = set(org["id"] for org in response_json)
 
-            self.diff_urls.append(github_event["pull_request"]["diff_url"])
+            self.diff_urls.append(
+                f"https://api.github.com/repos/{GITHUB_REPOSITORY}/"
+                f"compare/master...{self.head_ref}"
```

Comment:
I missed it at the review. We need to use `github_event["pull_request"]["head"]["label"]` here